tests/test_server_basic.py

```diff
@@ -1526,4 +1526,4 @@ def test_param_value(dispatcher_live_fixture):
     print("content:", c.text)
     jdata=c.json()
 
-    assert jdata['products']['echo']['ang'] == 2.0
\ No newline at end of file
+    assert jdata['products']['echo']['ang'] == 0.0005555555555555556
```

Comment:
But my ultimate goal was to have 2.0 preserved here

Reply:
As discussed in #280 , we decided to set the all parameters to their default value.

This discussion started with #227 for the Time parameter, but we forgot to consider the case you just encountered.
In any case we agreed to set by default all the parameters to their default units.

Reply:
OK, finally I grasp the reason it is changed to work this way.

Reply:
check now, I might have found a better way, but perhaps some more testing is required